README.md

```diff
@@ -362,6 +363,9 @@ between **0 and 10**. A confidence of 0 indicates that the check was unable to
 achieve any real signal, and that the result should be ignored. A confidence of 10
 indicates the check was completely sure of the result. 
 
+### Using public data
+[example of using scorecard bigquery](examples/bigquery/scorecarddata)
```

Comment:
How about we link to `naveensrinivasan/scorecarddata` repo here? Since this is a repo of its own, IMO we should maintain this repo separately rather than including it in scorecard repo.

Reply:
Sounds good. I will do a PR for this.